### ISSUE TITLE
[Feature Request] Restore OLD hooking to player voting

### ISSUE THREAD

**Issue by timbergeron** [quakeone/crmod-plus#28]
Restore OLD hook onto player doesn't let go behavior or make it a var

**Comment by inf1niti** [comment]
I agree with this, but I could see people having feelings about it either way.  It would be such a weird thing to vote though, lol.

**Comment by timbergeron** [comment]
Server admin setting 

--
Tim Bergeron
timbergeron.com
